tests/test_photon_ops.py

```diff
@@ -682,7 +683,7 @@ def test_phase_affects_image():
 
     # Now add some phase and process again
     config = galsim.config.CleanConfig(config)
-    config.update(**TEST_BASE_CONFIG) # restore _icrf_to_field
+    config.update(**deepcopy(TEST_BASE_CONFIG)) # restore _icrf_to_field
```

Comment:
The canonical way to do this, rather than deepcopy is `config = galsim.config.CleanConfig(config)` whenever you change a dict and need to reset the compiled bits.